doc/input-output-reference/src/overview/group-schedules.tex

```diff
@@ -542,6 +542,26 @@ \subsubsection{Inputs}\label{inputs-9-019}
   60;                 !- Minutes per Item
 \end{lstlisting}
 
+\paragraph{Field: Adjust Schedule for Daylight Savings}\label{adjust-schedule-for-daylight-savings-1}
+
+This field indicates whether or not to shift the schedule forward an hour during daylight savings time. If "Yes" the schedule skips forward an hour in the CSV on the "spring forward" day and repeats an hour on the "fall back" day. If  "No", the schedule is read and applied in standard time, without adjusting for daylight savings time. This indicator is defaulted to "Yes" if no input is provided. 
+
```

Comment:
Maybe add a reference to the [DST](https://bigladdersoftware.com/epx/docs/9-6/input-output-reference/group-location-climate-weather-file-access.html#runperiodcontroldaylightsavingtime) object?

Reply:
DST object referenced in LaTeX.